### DISCUSSION TITLE
How to get started with contribution

### DISCUSSION THREAD

**Discussion by harshkunz** [cncf/mentoring#1082] [Q&A] [answered]
Respected sir/madam,

I am Harsh Pratap, a Undergrad. I have just entered my 2nd year at IIIT Jabalpur. I am new in open source contribution but I am well aware of Python, C++, JavaScript and node.js. I would love to contribute to your Organization but could you please tell me how to get started ?
Hoping to hear from you soon.
Regards
Harsh

**Comment by aliok** [comment, marked as answer]
Hi @harshkunz 

I would recommend reading this guide: https://contribute.cncf.io/contributors/getting-started/
